aip/general/0003.md

```diff
@@ -0,0 +1,34 @@
+---
+id: 3
+state: approved
+created: 2023-03-28
+placement:
+  category: meta
+  order: 30
+---
+
+# AIP Versioning
+
+This AIP defines the versioning scheme of the AIPs.
+
+## Guidance
+
+- The AIPs **must** be versioned by date, using the [ISO-8601 format of `YYYY-MM-DD`](https://www.w3.org/TR/NOTE-datetime), that corresponds to the date the version was added.
```

Comment:
I thought the proposal was `yyyy-MM`--not to include day of month.

Reply:
please see the issue comments: @jskeet suggested ISO-8601 `YYYY-MM-DD`.

Reply:
Note this is also stated in the appendix.